src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+
+  --color-for-main-text: #060B35;
+  --color-for-secondary-text: #616070;
+  --color-for-button: #00ACDC;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 33px 19px 16px;
+  box-sizing: border-box;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__h1 {
+    margin: 0;
+    margin-bottom: 4px;
+    max-width: 166px;
+    color: var(--color-for-main-text);
+    font-size: 12px;
+    font-style: normal;
```

Comment:
this rule is default, no needs to rereclare. check other cases
```suggestion
```

Reply:
And here